opi/plugins/yandex-disk.py

```diff
@@ -0,0 +1,23 @@
+import opi
+from opi.plugins import BasePlugin
+import subprocess
+
+class YandexDisk(BasePlugin):
+	main_query = "yandex-disk"
+	description = "Yandex.Disk cloud storage client"
+	queries = ('yandex-disk')
+
+	@classmethod
+	def run(cls, query):
+		if not opi.ask_yes_or_no("Do you want to install yandex-disk from yandex-disk repository?", 'y'):
+			return
+
+		opi.add_repo(
+			filename = 'yandex-disk',
+			name = 'yandex-disk',
+			url = 'https://repo.yandex.ru/yandex-disk/rpm/stable/$basearch/',
+			gpgkey = 'https://repo.yandex.ru/yandex-disk/YANDEX-DISK-KEY.GPG'
```

Comment:
While there is a key file lying around in the repo, there is no signed `repomd.xml` which would make zypper complain.
So disable the gpg check here (while still importing the gpg key - just in case it was eg. used to sign the actual rpm).

```suggestion
			gpgkey = 'https://repo.yandex.ru/yandex-disk/YANDEX-DISK-KEY.GPG',
			gpgcheck = False
```

Note that this will only work with the newest master as I just found out right now that the `gpgcheck` argument of `add_repo()` was broken and fixed it in https://github.com/openSUSE/opi/commit/5ff1ffcc7a81835d68f83c42eef5fd4b6a17076f.